src/advanced/bitcoin-anchoring.md

```diff
@@ -168,24 +168,25 @@ anchoring transaction only if the previous anchoring chain failed
   anchoring transaction and the hash of the corresponding Exonum block. The
   first anchoring transaction in the chain uses the UTXO of the funding
   transaction
-- Every validator creates an Exonum transaction containing a new proposed
-  anchoring transaction and its vote for said transaction. The validators commit
-  these transactions to the Exonum blockchain
+- Every validator node has an external sync [util][btc_anchoring_sync] that
```

Comment:
Nit: Here and in some other places, you use "validator" - although, if I understand correctly, anchoring is no longer performed necessarily by the blockchain validators. I'd use introduce a term like "anchoring node" and use it instead.